egs/reverb/asr1/run.sh

```diff
@@ -91,16 +96,30 @@ set -o pipefail
 
 train_set=tr_simu_8ch_si284
 train_dev=dt_mult_1ch
-recog_set="dt_real_1ch dt_simu_1ch et_real_1ch et_simu_1ch"
+recog_set="dt_real_1ch dt_simu_1ch et_real_1ch et_simu_1ch dt_real_2ch_beamformit dt_simu_2ch_beamformit et_real_2ch_beamformit et_simu_2ch_beamformit dt_real_8ch_beamformit dt_simu_8ch_beamformit et_real_8ch_beamformit et_simu_8ch_beamformit dt_real_1ch_wpe dt_simu_1ch_wpe et_real_1ch_wpe et_simu_1ch_wpe dt_real_2ch_wpe dt_simu_2ch_wpe et_real_2ch_wpe et_simu_2ch_wpe dt_real_8ch_wpe dt_simu_8ch_wpe et_real_8ch_wpe et_simu_8ch_wpe dt_cln et_cln"
```

Comment:
You don't need all experiments. You can just perform WPE+beamformit

Reply:
1ch with WPE is not needed? "WPE+beamformit" for both 2ch and 8ch? what about the clean - shall I remove that too?

Reply:
Of course you need 1c with WPE, but others are not necessary. Yes, you should also remove the clean one.

Reply:
Sure, this is changed